src/providers/WorkflowCore.Persistence.MongoDB/Services/MongoPersistenceProvider.cs

```diff
@@ -44,7 +45,8 @@ static MongoPersistenceProvider()
                 x.MapProperty(y => y.WorkflowDefinitionId);
                 x.MapProperty(y => y.Version);
                 x.MapProperty(y => y.NextExecution);
-                x.MapProperty(y => y.Status);
+                x.MapProperty(y => y.Status)
+                    .SetSerializer(new EnumSerializer<WorkflowStatus>(BsonType.String));
```

Comment:
is this backward compatible with previously persisted workflows?

Reply:
yes, it is